core/dbt/events/base_types.py

```diff
@@ -88,6 +101,29 @@ def get_pid(self) -> int:
             self.pid = os.getpid()
         return self.pid
 
+    def get_node_info(self):
+        return None
```

Comment:
this doesn't look right.

Reply:
what is this needed for?

Reply:
@nathaniel-may right now in the `event_dict` I have `'node_info': e.get_node_info() ` so without this you trigger an Attribute Error.  There's a comment in `functions.py` that this will go away after I can leverage `event_to_serializable_dict` in #4326.

Reply:
ok perfect